app/services/preserved_object_handler.rb

```diff
@@ -32,29 +32,32 @@ def initialize(druid, incoming_version, incoming_size, endpoint)
     @logger = PreservationCatalog::Application.logger
   end
 
-  def create_after_validation
+  def create_after_validation(checksums_validated = false)
     results.check_name = 'create_after_validation'
     if invalid?
       results.add_result(AuditResults::INVALID_ARGUMENTS, errors.full_messages)
     elsif PreservedObject.exists?(druid: druid)
       results.add_result(AuditResults::DB_OBJ_ALREADY_EXISTS, 'PreservedObject')
     elsif moab_validation_errors.empty?
-      create_db_objects(PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      creation_status = (checksums_validated ? PreservedCopy::OK_STATUS : PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      create_db_objects(creation_status, checksums_validated)
     else
-      create_db_objects(PreservedCopy::INVALID_MOAB_STATUS)
+      create_db_objects(PreservedCopy::INVALID_MOAB_STATUS, checksums_validated)
     end
 
     results.report_results
   end
 
-  def create
+  def create(checksums_validated = false)
     results.check_name = 'create'
     if invalid?
       results.add_result(AuditResults::INVALID_ARGUMENTS, errors.full_messages)
     elsif PreservedObject.exists?(druid: druid)
       results.add_result(AuditResults::DB_OBJ_ALREADY_EXISTS, 'PreservedObject')
     else
-      create_db_objects(PreservedCopy::VALIDITY_UNKNOWN_STATUS)
+      creation_status = (checksums_validated ? PreservedCopy::OK_STATUS : PreservedCopy::VALIDITY_UNKNOWN_STATUS)
```

Comment:
Since we use the ternary in more than one place, do we want to wrap it up in its own method because DRY?

Reply:
i think i'd be inclined to leave it as-is, just because i find it pretty readable, and i can't think of a good method name, but wouldn't be totally opposed to it (esp if you had a good method name in mind)

Reply:
If `creation_status(checksums_valid)` isn't a good enough method signature, I'd be OK leaving it as is for now, maybe with a ticket or TODO noting the opportunity to refactor.

Reply:
i think i call it "new_status" when it is used by update_validation.   Perhaps we can briefly decide together if we want to ticket this or let it ride.

Reply:
I ticketed as #991 and put in backlog column.